cmd/edit/cluster/cmd.go

```diff
@@ -19,6 +19,8 @@ package cluster
 import (
 	"errors"
 	"fmt"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/openshift/moactl/pkg/confirm"
```

Comment:
Please move these imports: the first one to the middle section (where third-party imports are) and the second one to the bottom section with the internal imports.